### PR TITLE
Added more context on Sysdig Secure

### PR DESCRIPTION
**Pull request by yathi-sysdig**
**What type of PR is this?**

| Uncomment one (or more) /kind <> lines:

/kind documentation

**Any specific area of the project related to this PR?**

Uncomment one (or more) /area <> lines:

/area integrations

**What this PR does / why we need it:**

**Which issue(s) this PR fixes:**

No issues, just adding to adopters list.

```release-notes
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Adopters
 
+Sysdig created Falco in 2016 to detect unexpected or suspicious activity using a rules engine on top of the data that comes from the sysdig kernel system call probe.
```
Comment (fntlnz) [inline ADOPTERS.md]: This is an unnecessary detail. This file only purpose is here to list adopters, not to tell the story of the project or what it does.

File: ADOPTERS.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+Sysdig provides tooling to help with vulnerability management, compliance, detection, incident response and forensics in Cloud-native environments. Sysdig Secure has extended falco to include: a rule library, the ability to update macros, lists & rules via the user interface and API, automated tuning of rules, and rule creation based on profiling known system behavior. On top of the basic Falco rules, Sysdig Secure implements the concept of a "Security policy" that can comprise several rules which are evaluated for a user-define infrastructure scope like Kubernetes namespaces, OpenShift clusters, deployment workload, cloud regions etc.
```
Comment (fntlnz) [inline ADOPTERS.md]: I'm totally fine in adding this, but here is the wrong location, please move it inside the list in alphabetical order.

File: ADOPTERS.md (hunk 1 above)
Comment (leodido) [inline ADOPTERS.md]:
```suggestion
```

**Comment by leodido** [comment]
Furthermore please edit the PR body using the PR template.

File: ADOPTERS.md (hunk 2 above)
Comment (yathi-sysdig) [reply ADOPTERS.md]: Corrected

**Comment by leodido** [comment]
Thanks for updating it Yathi 🤗 

Reviewing tomorrow morning!

/hold

File: ADOPTERS.md (hunk 2 above)
Comment (fntlnz) [reply ADOPTERS.md]: Thanks!

**Pull request merged**